src/base/bittorrent/tracker.cpp

```diff
@@ -29,15 +29,23 @@
  * Contact : chris@qbittorrent.org
  */
 
+#include <limits>
 #include <vector>
 #include <libtorrent/bencode.hpp>
 #include <libtorrent/entry.hpp>
 
+#include <boost/asio/ip/address.hpp>
+#if defined(BOOST_ASIO_DYN_LINK)
+#include <boost/asio/impl/src.hpp>
```

Comment:
I mean you shouldn't include `<boost/asio/impl/src.hpp>` here. If you can't build qBittorrent without this, you've incorrectly set up your build environment.